definitions/npm/lodash_v4.x.x/flow_v0.63.x-/lodash_v4.x.x.js

```diff
@@ -260,10 +260,10 @@ declare module "lodash" {
       fromIndex?: ?number
     ): -1;
     // alias of _.head
-    first<T>(array: ?$ReadOnlyArray<T>): T;
+    first<T>(array: ?$ReadOnlyArray<T>): T | void;
     flatten<T, X>(array?: ?Array<Array<T> | X>): Array<T | X>;
     flattenDeep<T>(array?: ?(any[])): Array<T>;
-    flattenDepth(array?: ?(any[]), depth?: ?number): any[];
```

Comment:
I wonder if this would work?
```js
flattenDepth(array: (any[]), depth?: ?number): any[];
flattenDepth(array?: ?(any[]), depth?: ?number): ?any[];
```
This would be ideal, because it wouldn't force null checks in cases where the `array` passed is guaranteed to be defined, but would require null checks otherwise.

Reply:
nevermind, I'm forgetting that lodash always returns an array for cases like this